pkgs/desktops/gnome/misc/gnome-panel/modulesdir-env-var.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/data/pkgconfig/libgnome-panel.pc.in b/data/pkgconfig/libgnome-panel.pc.in
+index c0e90b90d..8d5d490ad 100644
+--- a/data/pkgconfig/libgnome-panel.pc.in
++++ b/data/pkgconfig/libgnome-panel.pc.in
+@@ -4,7 +4,6 @@ libdir=@libdir@
+ includedir=@includedir@
+ datarootdir=@datarootdir@
+ layoutsdir=@datadir@/gnome-panel/layouts
+-modulesdir=@libdir@/gnome-panel/modules
```

Comment:
Is not this used by packages so that they know where to install files to?

Reply:
The idea with modulesdir is that applet packages should install their applets there (like `/usrlib/gnome-panel/...`), but this doesn't work on Nix. I decided to replace it with a bogus path so that applet packages fail to build unless they handle this.

Reply:
Ideally, packages would install to their own prefix as described in https://www.bassi.io/articles/2018/03/15/pkg-config-and-paths/.

Reply:
OK, I left it in.

Reply:
I would just keep the file as is.

Reply:
Done.